### PR TITLE
commonLabels value for chart

### PR DESCRIPTION
**Pull request by jklaw90**
Adding the ability to add common labels for resources. Currently we can't use the chart as is since we require extra custom labels.

### CODE REVIEW

**Comment by jklaw90** [comment]
I signed it

File: charts/descheduler/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The following table lists the configurable parameters of the _descheduler_ chart
 | `image.repository`             | Docker repository to use                                                                                              | `k8s.gcr.io/descheduler/descheduler` |
 | `image.tag`                    | Docker tag to use                                                                                                     | `v[chart appVersion]`                |
 | `image.pullPolicy`             | Docker image pull policy                                                                                              | `IfNotPresent`                       |
-| `imagePullSecrets`             | Docker repository secrets                                                                                              | `[]`                       |
+| `imagePullSecrets`             | Docker repository secrets                                                                                             | `[]`                                 |
```
[the remark this answers is not in the document]
Comment (jklaw90) [reply charts/descheduler/README.md:52]: i think vscode is auto formatting it so it lines up with the other ends
here's a screenshot without the space
image

**Review by damemi — COMMENTED** [review]
/approve

File: charts/descheduler/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (damemi) [reply charts/descheduler/README.md:52]: that's fine, it was just a nit, not a huge issue

**Comment by jklaw90** [comment]
do i write the lgtm?
/lgtm

**Pull request merged**